src/libraries/System.Private.CoreLib/src/System/Buffers/Text/Utf8Parser/Utf8Parser.Boolean.cs

```diff
@@ -12,16 +12,14 @@ public static partial class Utf8Parser
         /// </summary>
         /// <param name="source">The Utf8 string to parse</param>
         /// <param name="value">Receives the parsed value</param>
-        /// <param name="bytesConsumed">On a successful parse, receives the length in bytes of the substring that was parsed </param>
-        /// <param name="standardFormat">Expected format of the Utf8 string</param>
+        /// <param name="bytesConsumed">On a successful parse, receives the length in bytes of the substring that was parsed</param>
+        /// <param name="standardFormat">Expected format of the Utf8 string. Supported formats are G and l.</param>
```

Comment:
If we're going to list formats, it should likely also call out that `default` is supported as well.

Reply:
I updated the comment to indicate that the default value is also accepted.